tests/nft/nft.cc

```diff
@@ -3157,7 +3157,7 @@ TEST_CASE("mata::nft::project_out(repeat_jump_symbol = false)") {
         Nft atm(delta, { 0 }, { 3 }, { 0, 1, 2, 0 }, 3);
 
         SECTION("project 0") {
-            Nft proj0 = project_out(atm, 0, false);
+            Nft proj0 = project_out(atm, OrdVector<Level>{ 0 }, false);
```

Comment:
What is the idea behind calling `project_out(const Nft& aut, const utils::OrdVector<Level>& levels_to_project, const bool repeat_jump_symbol = true);` insted of `mata::nft::project_out(const Nft& aut, const Level level_to_project, const bool repeat_jump_symbol)`?

Reply:
There is no difference in functionality. I just wanted to test both versions of the function: one where a single level is called, and one where a vector containing only the single level is called.

Reply:
Thah is a good idea.